### ISSUE TITLE
use-t with macros

### ISSUE THREAD

**Issue by novemberborn** [avajs/eslint-plugin-ava#127]
I have a macro test which takes a function argument:

``` js
test('with the error if fn throws', asapInvokesCrashHandler, err => { throw err })
```

`use-t` complains about `err` not being `t`. I think it should check whether it is the first argument, or if it's the second, whether the first argument could be a string. That should prevent false positives.
